manifests/kustomize/gcp-workload-identity-setup.sh

```diff
@@ -21,7 +21,7 @@ SYSTEM_GSA=${SYSTEM_GSA:-$CLUSTER_NAME-kfp-system}
 USER_GSA=${USER_GSA:-$CLUSTER_NAME-kfp-user}
 
 # Kubernetes Service Account (KSA)
-SYSTEM_KSA=(ml-pipeline-ui)
```

Comment:
Visualization server is currently using default service account.
You need to add a specific service account for it first.

You can follow examples here:
https://github.com/kubeflow/pipelines/blob/10a4f55e3ae5ed0e47466ec3bd4f0d56b9333cec/manifests/kustomize/base/pipeline/ml-pipeline-viewer-crd-deployment.yaml#L27
https://github.com/kubeflow/pipelines/blob/10a4f55e3ae5ed0e47466ec3bd4f0d56b9333cec/manifests/kustomize/base/pipeline/ml-pipeline-viewer-crd-sa.yaml#L4

Reply:
Reminder, also update the same change in hosted manifests: https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml#L444

Reply:
done